README.md

```diff
@@ -6,7 +6,7 @@
   <br />
   <p>
     <a href="https://horion.download/injector"><img src="https://img.shields.io/static/v1?label=download&message=latest&color=12c970&logo=docusign&logoColor=white" alt="Download Injector" /></a>
-    <a href="https://horion.download/discord"><img src="https://img.shields.io/discord/732833913705201736?color=5865F2&logo=discord&logoColor=white" alt="Discord server" /></a>
```

Comment:
No, the old link can be easily changed to a new server.

Reply:
the old link is the new server